packages/search/tests/unit/services/navi-search-provider-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | navi-search-provider', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:navi-search-provider');
+    assert.ok(service);
+  });
+
+  test('get all search providers', function(assert) {
+    let service = this.owner.lookup('service:navi-search-provider');
+    let availableSearchProviders = service.all();
+    assert.ok(availableSearchProviders.length > 0);
```

Comment:
maybe this should assert on the exact number? 
`availableSearchProviders.length  == 1`
that way if we add a new one it'll fail. We'd have to update the tests to cause it to pass? Let me know what you think

Reply:
`assert.equal(availableSearchProviders.length, 1, "Put some assert message here so it's easier to debug if it fails");`